_life/money-americans.md

```diff
@@ -0,0 +1,406 @@
+in the US, with offers like "3% cashback on all groceries" or "$200 cashback if you
+spend $1,000 in the first 3 months". Because credit cards are so ubiquitous, Starbucks
+sets their prices to account for the interchange fees they will have to pay for the
+predictable portion of transactions that will happen via credit card. Thus it is
+sound financial advice in the US to get and use a credit card because if you don't
+then you are just paying for cashback for those who do.[^4]
+
+So what about Europe? (Including Switzerland for this paragraph.) Well, EU financial
+regulators capped the interchange fee at 0.3% in 2015, citing a variety of reasons:
+the Single Market philosophy, leveling the playing field for innovation, fairness for
+retailers, the fact that rewards move money from the poor to the rich, etc. (not all
+of which I agree with). Compare that to US interchange fees which are typically in
+the 1&ndash;3% range. Consequently, credit card rewards programs are meager in
+Europe, and cashback if it exists is no more than 0.3% for cards without an annual
+fee. You will notice an accompanying striking lack of credit card marketing and much
+lower usage of credit cards.[^5]
+
+In short, getting a credit card in Switzerland is hardly worth the effort&mdash;if
+they are even willing to give you one. The basic, no-fee card that UBS offers to
+existing customers is granted if the customer either:
+
+ * demonstrates an employment contact valid for at least two years, which an EPFL PhD
+   student cannot do, because our contracts are renewed yearly; or
+ * makes a 'locked' deposit equal to the credit limit on the card that can be
+   withdrawn only when the card is closed. (What's the point, then?)
+   
+However, if you have a US-issued credit card with no foreign transaction fees but
+good rewards and some amount of liquid money in the US that you want to transfer to
+Switzerland anyway, then it's a great idea to use that card (and pay it off with said
+liquid money) for everyday purchases in Switzerland. You're essentially earning
+US-level cashback on the USD-CHF transfer which you would otherwise have to pay a
+financial institution for, and credit cards usually have better exchange rates than
+banks. But the card issuer and network aren't making much money from interchange and
+nor can they sell you as an advertizing target to American retailers. So you're
+eating into the profits of Big Banking! Go you! At least until they close your card
+because they notice you've permanently moved out of the country.
+
+_Personal note from Shardul:_ I actually did the above with a Wells Fargo Autograph
+card, which is a Visa Signature card that gives 1% cashback on all purchases, plus 2%
+on travel, dining, and streaming subscriptions, plus $200 if you spend $1,000 in the
+first 3 months, which I did by booking my tickets to Switzerland and buying a bike
+after I got here. They haven't closed it yet!
```

Comment:
This section is very long and detailed but for a rather short actionable message: credit card cashbacks are much lower in Europe than the US due to various regulations, thus it's not worth getting a credit card in Europe, but using a US card is worth it. Or is there something more?

Reply:
By the way, one way for everyone to get a "credit" card here is to get a prepaid one. I don't know if it's a very interesting value proposition though, do they even have cashbacks?

Reply:
Yes, at least my UBS-issued one does (0.2%), but I don't see the value. And you are right that the actionable message is very short but the inner workings of American credit cards are fascinating to me so I included it. Splitting into 'actionable' and 'context' makes sense here too.